testutil/keeper/unit_test_helpers.go

```diff
@@ -207,6 +209,17 @@ func GenPubKey() (crypto.PubKey, error) {
 	return cryptocodec.ToTmPubKeyInterface(privKey.PrivKey.PubKey())
 }
 
+func GetClientState(chainID string) *ibctmtypes.ClientState {
```

Comment:
Could these methods be named `GetDefaultClientState` and `GetDefaultConsensusState` respectively, or do they serve a different function than returning populated defaults?